src/components/blog-item.tsx

```diff
@@ -0,0 +1,39 @@
+import Image from 'next/image'
+import Link from 'next/link'
+
+import blogStyles from '../styles/blog.module.css'
+
+interface blogProps {
+  image?: string
+  title?: string
+  summary?: string
+  datePublished?: string
+  author?: string
+}
+
+const blogItem = ({
```

Comment:
```suggestion
const BlogItem = ({
```

[Components should be PascalCase](https://reactjs.org/docs/jsx-in-depth.html#user-defined-components-must-be-capitalized) (I would also rename the file to match the component name for consistency)